### ISSUE TITLE
Improve UI when user exceeds free trial requests

### ISSUE THREAD

**Issue by djarran** [trypear/pearai-app#315]
## Describe the bug
JSON output is shown when user has exceeded free trial requests. We'd want to handle this error response for end users.

## Screenshots

![CleanShot 2024-09-09 at 19 14 09@2x](https://github.com/user-attachments/assets/82882377-f79a-4cb5-ae8e-262ae6dfa87c)

**Comment by darkstar331** [comment]
@Fryingpannn ca u assign this issue to me?

**Comment by Fryingpannn** [comment]
@darkstar331 yes. did you apply to the quest by any chance? also what's ur tag in discord?

**Comment by Fryingpannn** [comment]
@darkstar331 Did you already start this? Currently about to assign it to someone else if not! (rgz)

**Comment by ssparser** [comment]
hello @Fryingpannn is this issue resolved?

**Comment by Dhoundiyal-Anshul** [comment]
can u assign this issue to me? @Fryingpannn
